consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/messagedata/QbftV1.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.consensus.qbft.messagedata;
+
+/** Message codes for iBFT v2 messages */
+public class QbftV1 {
+  public static final int PROPOSAL = 0;
+  public static final int PREPARE = 1;
+  public static final int COMMIT = 2;
+  public static final int ROUND_CHANGE = 3;
+
```

Comment:
nit: unnecessary space.

Reply:
kinda like the space - the numbers above are the msgs we use, the MESSAGE_SPACE is a slightly different concept.

Reply:
in that case, its alright.